androidartifact/apk_info.go

```diff
@@ -65,8 +65,13 @@ func parsePackageField(aaptOut, key string) string {
 }
 
 // ParsePackageInfos parses package name, version code and name from `aapt dump badging` command output.
-func ParsePackageInfos(aaptOut string) (string, string, string) {
-	return parsePackageField(aaptOut, "name"),
+func ParsePackageInfos(aaptOut string, isAAB bool) (string, string, string) {
```

Comment:
There is a seperate parseAPKInfo function, is the isAAB paramter needed, or it is always true?

Reply:
We use two functions for the apk parsing which are `parseAPKInfo` and `getAPKInfoWithAapt` one is the fallback for the other:
https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/d00d27bb9961bd22db866568f775a32d975abfe7/androidartifact/apk_info.go#L173-L183

So we need the `isAAB` parameter to differentiate the apk and aab case.